osv/ecosystems/alpine.py

```diff
@@ -141,14 +141,17 @@ def _get_versions(branch: str, package: str) -> typing.List[str]:
 
     relative_path = os.path.relpath(directories[0], checkout_dir)
 
-    regex_test = '\\|'.join([
-        '\\(' + x.removeprefix('+') + '\\)'
-        for x in Alpine._PKGVER_ALIASES + Alpine._PKGREL_ALIASES
-    ])
+    regex_test_aliases = '\\|'.join(
+        ['\\(' + x.removeprefix('+') + '\\)' for x in Alpine._PKGVER_ALIASES])
 
+    regex_test_revisions = '\\|'.join(
+        ['\\(' + x.removeprefix('+') + '\\)' for x in Alpine._PKGREL_ALIASES])
+
+    # https://git-scm.com/docs/git-log/2.33.1#Documentation/git-log.txt--Lltstartgtltendgtltfilegt
     stdout_data = subprocess.check_output([
         'git', 'log', '--oneline', '-L',
-        '/' + regex_test + '/,+1:' + relative_path
+        '^/' + regex_test_aliases + '/,+1:' + relative_path, '-L',
+        '^/' + regex_test_revisions + '/,+1:' + relative_path
```

Comment:
nit: Will the linter let you format this as
```
'git', 'log', '--oneline',
'-L', '^/' + ...,
'-L', '^/' + ...,
],
```
So the -L flag is together with its value

Reply:
I tried, the linter keeps undoing my attempts :(